tests/test_roxarapi/test_roxarapi_reek.py

```diff
@@ -244,6 +244,31 @@ def test_rox_get_modify_set_grid(roxar_project):
     assert grd2.dimensions == grd1.dimensions
 
 
+@pytest.mark.requires_roxar
+def test_rox_get_modify_set_get_grid_with_subzones(roxar_project, roxinstance):
+    """Get, modify and set + get a grid from a RMS project using subzones/subgrids."""
+
+    grd = xtgeo.grid_from_roxar(roxar_project, GRIDNAME1)
+
+    zonation = OrderedDict()
+    zonation["intva"] = 4
+    zonation["intvb"] = 7
```

Comment:
I think maybe you want to squash 1f051a0 and dbb270934 do avoid having a test disappear and reappear.